Cargo.toml

```diff
@@ -1,9 +1,5 @@
 [workspace]
```

Comment:
Right, I forgot to undo that. I suppose that's the default in vscode or the "better toml" extension.

Reply:
Seems like something we could all install, I’ll see how it works on my end.

Reply:
I installed Even Better TOML, because Better TOML is deprecated. Here are my settings, I chose them to roughly match `rustfmt`:
```json
    "evenBetterToml.formatter.alignEntries": true,
    "evenBetterToml.formatter.alignComments": true,
    "evenBetterToml.formatter.allowedBlankLines": 1,
    "evenBetterToml.formatter.arrayTrailingComma": true,
    "evenBetterToml.formatter.arrayAutoExpand": true,
    "evenBetterToml.formatter.arrayAutoCollapse": true,
    "evenBetterToml.formatter.columnWidth": 100,
    "evenBetterToml.formatter.indentTables": true,
    "evenBetterToml.formatter.indentString": " ",
    "evenBetterToml.formatter.reorderArrays": true,
    "evenBetterToml.formatter.trailingNewline": true,
```

Reply:
I undid the unnecessary changes in this PR.

Reply:
Thanks, that will help with merging.